ironfish-cli/bin/ironfish

```diff
@@ -41,5 +41,10 @@ else
   if [ "$DEBUG" == "*" ]; then
     echoerr IRONFISH_BINPATH="$IRONFISH_BINPATH" "$NODE" "$DIR/run" "$@"
   fi
-  "$NODE" "$DIR/run" "$@"
+
+  if [ "$IRONFISH_EXPOSE_GC" == "true" ]; then
+    "$NODE" "--expose-gc" "$DIR/run" "$@"
+  else
+    "$NODE" "$DIR/run" "$@"
+  fi
 fi
```

Comment:
This is is a generated file, these changes will be lost if we re-generate the oclif project. However, it's fine if this is lost.